cli/logout.go

```diff
@@ -41,25 +42,46 @@ func (l *LogoutCommand) runLogout(args []string) error {
 	}
 
 	// TODO: better way to get registry server address, or get it from a default variable
-	registry := serverAddress
-	if registry == "" {
-		registry = "https://index.docker.io"
+	var (
+		registry = serverAddress
```

Comment:
I think `serverAddress ` and `registry ` is redundant. We can only use one. WDYT?